parser/src/aspects/call.rs

```diff
@@ -226,36 +240,17 @@ impl<'a> CallAspect<'a> for Parser<'a> {
     fn may_be_at_programmatic_call_start(&self) -> bool {
         self.cursor
             .lookahead(
-                repeat(of_type(Identifier).and_then(of_type(ColonColon)))
+                of_type(TokenType::Reef)
+                    .and_then(of_type(TokenType::ColonColon))
+                    .then(repeat(
+                        of_type(TokenType::Identifier).and_then(of_type(TokenType::ColonColon)),
+                    ))
```

Comment:
```suggestion
                repeat(
                    of_types(&[TokenType::Identifier, TokenType::Reef])
                        .and_then(of_type(TokenType::ColonColon)),
                )
```
Let the actual path validator decides if the path is actually valid.